ui-library-add-call-recording/api/local.settings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "IsEncrypted": false,
+  "Values": {
+    "FUNCTIONS_WORKER_RUNTIME": "dotnet",
+    "ACS_CONNECTION_STRING": "ENTER_ACS_CONNECTION_STRING"
```

Comment:
In general we never want to use the acronym "ACS". We went through an exercise of scrubbing this acronym out of our code and docs. I would suggest sticking to what we had in our hero samples or to find a more suitable const name

Reply:
Done.